packages/casting/src/follower-cosmjs.js

```diff
@@ -403,53 +387,43 @@ export const makeCosmjsFollower = (
    * @yields {ValueFollowerElement<T>}
    */
   async function* getLatestIterable() {
-    let blockHeight;
-    let data;
+    let lastData;
     for (;;) {
-      const currentBlockHeight = await getBlockHeight();
-      if (currentBlockHeight === blockHeight) {
-        // TODO Long-poll for next block
-        // https://github.com/Agoric/agoric-sdk/issues/6154
-        await E(leader).jitter(where);
-        continue;
-      }
```

Comment:
This check is intended to prevent the iterable from stuttering duplicate values if the consumer side polls faster than the chain advances. Without it, we fall thru to the check below it that discards duplicates if the data are bytewise identical, which should have the same effect but leans on a code path that we might remove if we can ever drop support for legacy cells. We should at least update the comment on line 407 which is no longer true.

Reply:
Ah. PTAL at https://github.com/Agoric/agoric-sdk/pull/6607/commits/49e8fddb7da6ebb56dd39eaaec12a4e646becc65